x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -138,7 +138,7 @@ export const DocumentCountChart: FC<Props> = ({
 
   return (
     <div
-      style={{ width: width ?? '100%', height: 120, display: 'flex', alignItems: 'center' }}
+      style={{ width: width ?? '100%', display: 'flex', alignItems: 'center' }}
```

Comment:
did you try using the `EuiFLexGroup` component?

Reply:
Updated here [`2d35742` (#147137)](https://github.com/elastic/kibana/pull/147137/commits/2d35742e343066904b17034fa60a0b1c3fb5536e)